src/types/Quality.ts

```diff
@@ -3,21 +3,27 @@ export const Quality = {
     key: "?",
     level: 0,
     text: "Unknown",
+    info: "The certificate quality couldn't be determined.",
   },
   DomainValidated: {
     key: "DV",
     level: 1,
     text: "Domain Validated",
+    info: "The certificate authority verifies domain-ownership.",
   },
   OrganizationValidated: {
     key: "OV",
     level: 2,
     text: "Organization Validated",
+    info:
+      "The certificate authority verifies domain-ownership and performs background checks on the organization.",
   },
   ExtendedValidated: {
     key: "EV",
     level: 3,
     text: "Extended Validated",
+    info:
+      "The certificate authority verifies ownership of the certificate's domain and organization.",

```

Comment:
```suggestion
      "The certificate authority verifies ownership of the certificate's domain and performs extensive background checka on the organization.",

```

Reply:
Maybe add link to https://cabforum.org/about-ev-ssl/ ?

Reply:
You wouldn't be able to click on this link, since the tooltip is only visible while you hover over the text or question mark icon.